core/src/main/resources/lib/form/repeatableProperty.jelly

```diff
@@ -35,6 +35,12 @@ THE SOFTWARE.
             <f:repeatableDeleteButton />
           </div>
         </f:block>
+
+    Due to a bug in Stapler data binding the model elements are only set if they consist of one or more values.
+    If all values have been removed in the user interface (i.e. the associated form is empty), then the setter is not
+    invoked anymore. As a workaround, you need to override the corresponding configure method and clear the model
+    property manually before invoking the data binding. See warnings-ng-plugin PR#266.
```

Comment:
is it worth linking directly to the PR if someone actually wants to see the detail?

Reply:
I'm not sure where this documentation is shown? I always use it as it is in IntelliJ source code view.

Reply:
It's published here:
https://reports.jenkins.io/core-taglib/jelly-taglib-ref.html#form:repeatableHeteroProperty
But I assume mostly consumed in IDEs

also see:
https://jenkins.io/doc/developer/development-environment/taglibs/

Reply:
Hmm, seems that I can't use links there...